### PR TITLE
Follow up for #1323 Batch Fulltext downloader

### PR DESCRIPTION
**Pull request by stefan-kolb**
- 	Rename menu item to plural for finding full texts
- 	Disable menu item when there are no documents selected for fulltext retrieval
- 	Fix layout of file dialog and increase size of file path significantly
old:
image

new:
image

### CODE REVIEW

File: src/main/java/net/sf/jabref/gui/externalfiles/FindFullTextAction.java [hunk 1]
```diff
@@ -41,12 +41,26 @@ public FindFullTextAction(BasePanel basePanel) {
 
     @Override
     public void init() throws Exception {
+        if (basePanel.getSelectedEntries().size() == 0) {
```
Comment (matthiasgeiger) [inline src/main/java/net/sf/jabref/gui/externalfiles/FindFullTextAction.java]: Wouldn't be a better approach to disable the menu item (as it is already done for various other actions)?

If selection size() is == 0 at this point you can just throw an exception.

Comment (koppor) [inline src/main/java/net/sf/jabref/gui/externalfiles/FindFullTextAction.java]: Refs #2027

Comment (stefan-kolb) [inline src/main/java/net/sf/jabref/gui/externalfiles/FindFullTextAction.java]: Not 100 % sure about this, but done.

File: src/main/java/net/sf/jabref/gui/JabRefFrame.java [hunk 2]
```diff
@@ -1591,6 +1594,9 @@ public void updateEnabledState() {
 
             boolean twoEntriesSelected = current.getSelectedEntries().size() == 2;
             setEnabled(twoEntriesOnlyActions, twoEntriesSelected);
+
+            boolean oneOrMoreEntriesSelected = current.getSelectedEntries().size() >= 1;
```
Comment (koppor) [inline src/main/java/net/sf/jabref/gui/JabRefFrame.java]: Why not checking for `!isEmpy()`?

File: src/main/java/net/sf/jabref/gui/externalfiles/FindFullTextAction.java [hunk 3]
```diff
@@ -41,26 +41,14 @@ public FindFullTextAction(BasePanel basePanel) {
...
     @Override
     public void init() throws Exception {
-        if (basePanel.getSelectedEntries().size() == 0) {
-            JOptionPane.showMessageDialog(
-                    basePanel,
-                    Localization.lang("Select at least one entry to perform this operation"),
-                    Localization.lang("No entry selected"),
-                    JOptionPane.ERROR_MESSAGE);
-            return;
+        if (basePanel.getSelectedEntries().size() > 0) {
```
Comment (koppor) [inline src/main/java/net/sf/jabref/gui/externalfiles/FindFullTextAction.java]: `!isEmpty()`, maybe add a logger.DEBUG statement?

**Pull request merged**